packages/monitoring/src/serviceMonitor/alivenessChecker.ts

```diff
@@ -0,0 +1,56 @@
+import { events } from "../helpers/events";
```

Comment:
I don't uderstand why we didn't do it like this


class ServiceMonitor {

    constructor(services: Array<IService>, interval) {
        ...
    }
    private checkLivenessOnce {

    }
    private checkHealthOnce {

    }
    monitor() {
        withInterval do your checkLivenessOnce and checkHealthOnce 
    }

}